### DISCUSSION TITLE
BeardLib 5.0

### DISCUSSION THREAD

**Discussion by Luffyyy** [diesel-modding/PAYDAY-2-BeardLib#608] [Ideas]
At some point I'd like to do BeardLib 5.0, if not one of the maintainers would do it.

Points of interest:

* Moving to a non-number based versioning (like 5.0.1)
* Pushing for a single folder, *soft* deprecation of the framework system (while still supporting mod_overrides and Maps). This folder being most likely the mods folder as most people as used to it and it's the most intuitive place for them. This is probably going to be done via ~~a SuperBLT plugin if including it natively in SBLT isn't an option.~~ Lua.
* Supporting Raid WW2

**Comment by segabl** [comment]
deprecation of the framework system means no more modular approach for features I assume? which kind of makes sense I guess as it doesn't really change anything for the end user.
